src/image/models/image.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('images')
+export class ImageEntity {
```

Comment:
Just as a reminder, because I am not sure if I remember that: how did we plan to store images? I'm just thinking we might need some kind of URL here 🤔

Reply:
Probably an assets directory or some cloud storage?
We can build the URL with `$ENV_VARIABLE/$IMAGE_TYPE/$IMAGE_ID`.
But I'm open to suggestions as this wasn't set in stone as far as I'm aware.

Reply:
Agreed - I don't think we should store any images as binary in the database. They should be packaged as project assets to the mobile apps - this will save us some space in the database, since such objects as images don't need database-specific stuff like relations. In case we wanted to show an external image (like a banner in the app) to the user, we could use a static cloud storage for this purpose.